src/vs/workbench/parts/extensions/browser/media/extensionActions.css

```diff
@@ -5,6 +5,8 @@
 
 .monaco-action-bar .action-item .action-label.extension-action {
 	padding: 0 5px;
+	outline-offset: 1px;
+	outline-width: 1.5px;
```

Comment:
Both outline offset and width of 2px is closer to what @misolori had shared in the issue